tensorflow/lite/python/lite.py

```diff
@@ -516,12 +516,19 @@ def convert(self):
 
     Raises:
       ValueError:
-        Multiple concrete functions are specified.
-        Input shape is not specified.
-        Invalid quantization parameters.
+        - No concrete functions is specified. when no concrete function
```

Comment:
nit:
1) Can you remove the dashes ("-")
2) Can you change the text to be simpler (you don't need to specify the text of the error in the docstring, just when the error can occur since the text can change over time):
```
No concrete functions is specified.
Multiple concrete functions are specified.
Input shape is not specified.
Invalid quantization parameters.
```

Reply:
I have made that change.